crypto/crypto.go

```diff
@@ -72,13 +72,15 @@ func (client *Crypto) Config() interface{} {
 }
 
 func (client *Crypto) setupFSBackend(config core.ServerConfig) error {
+	log.Logger().Infof("Setting up FileSystem backend for storage of private key material. Not allowed in strict mode, consider using the Vault backend.")
 	fsPath := path.Join(config.Datadir, "crypto")
 	var err error
 	client.Storage, err = storage.NewFileSystemBackend(fsPath)
 	return err
 }
 
 func (client *Crypto) setupVaultBackend(_ core.ServerConfig) error {
+	log.Logger().Infof("Setting up Vault backend for storage of private key material.")
```

Comment:
I'd say debug

Reply:
I agree, since if you accidentally filesystem backend (which you can't do accidentally in PRD, since strict mode enforces you to choose one) we write an INFO log informing you filesystem backend is enabled.